pyiron/lammps/control.py

```diff
@@ -208,32 +208,28 @@ def pressure_to_lammps(self, pressure, rotation_matrix):
             rotation_matrix (numpy.ndarray): The 3x3 matrix rotating from the pyiron to Lammps coordinate frame.
 
         Returns:
-            (numpy.ndarray): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
+            (list): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
         """
 
         # Get something with six elements
         if not hasattr(pressure, "__len__"):
-            if not np.isclose(np.matrix.trace(rotation_matrix), 3) and pressure == 0.0:
-                pressure = np.append(pressure * np.ones(3), pressure * np.ones(3))
+            if np.isclose(pressure, 0.0):
+                pressure = 6*[0.0]
             else:
-                pressure = np.append(pressure * np.ones(3), 3 * [None])
+                pressure = [float(pressure), float(pressure), float(pressure), None, None, None]
         else:
             if len(pressure) > 6:
                 raise ValueError("Pressure can have a maximum of 6 values, (x, y, z, xy, xz, and yz), but got " +
                                  "{}".format(len(pressure)))
-            all_pressures = 6 * [None]
-            all_pressures[:len(pressure)] = pressure
-            pressure = np.array(all_pressures, dtype=float)
```

Comment:
Sorry actually quite a lot of lines have been changed but wouldn't it have been enough to erase just `dtype=float`?

Reply:
ok I just tested it on my local branch and saw that the only one change you need is to remove `dtype=float`. You don't need anything else.